libcrux-ml-dsa/src/sample.rs

```diff
@@ -21,12 +21,12 @@ fn rejection_sample_less_than_field_modulus(
 
             let potential_coefficient = ((b2 << 16) | (b1 << 8) | b0) & 0x00_7F_FF_FF;
 
-            if potential_coefficient < FIELD_MODULUS && *sampled < COEFFICIENTS_IN_RING_ELEMENT {
+            if potential_coefficient < FIELD_MODULUS && *sampled < out.coefficients.len() {
```

Comment:
Why stop using the constant here?

Reply:
It just seemed a bit more DRY, plus we don't have to import the constant in anymore, not a big deal though I can revert it if need be.

Reply:
No I think this is fine. The compiler knows this is a constant.